src/Sequence.cc

```diff
@@ -419,6 +425,9 @@ Sequence::Iterator
 Sequence::execute_range(Iterator step_begin, Iterator step_end, Context& context,
                         CommChannel* comm)
 {
+    using std::chrono::milliseconds;
+    using std::chrono::duration;
+
```

Comment:
... `milliseconds` and `duration` are not even used in this function?

Reply:
Residue of adding sequence timeout. Removed.